src/ipldBin.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+const CID = require('cids')
+
+// binary resolver
+module.exports = {
+  resolver: {
+    multicodec: 'base2',
```

Comment:
I remember that discussed this in great length and the reasons why raw is different from base2. If I remember correctly, you pointed out that base2 is actually incorrect .

I'm not finding the issue where that discussion happened (I think it was in IRC, our bad for not logging it).

All of that said, let's use the codec `raw`  here just to avoid any future confusion https://github.com/multiformats/multicodec/blob/master/table.csv#L3

Reply:
Base2 codec is literal stream of ASCI: `010100111010110`.

Reply:
@Kubuxu does `base2` have a code yet? in the multiformat table it has the same value as `bin`